js/main.js

```diff
@@ -0,0 +1,85 @@
+/* Here's what the pine tree should look like when you specify a height of 7, and use the asterisk character.
+
+01        *
+03       ***
+05      *****
+07     *******
+09    *********
+11   ***********
+13  *************
+
+*/
+
+var pineTree = {
```

Comment:
FYI, creating this object here, in the global scope, serves no purpose because the only other place that you are referencing it is when you modify the property values in the `checkValidRun()` method. It's best to conserve that memory by defining the object inside the `checkValidRun()` method. This will allow the object to be wiped from your computer's memory once your code completely executes.